Taskfile.yml

```diff
@@ -78,7 +78,7 @@ tasks:
       - >
         docker-compose up --always-recreate-deps --force-recreate --remove-orphans --renew-anon-volumes --timeout=0 --detach
         --build
-        postgres tigris mongodb
+        postgres tigris cockroach mongodb 
```

Comment:
```suggestion
        postgres tigris cockroach mongodb
```

Reply:
That's weird I've checked for trailing spaces